src/app/home/home.component.html

```diff
@@ -12,8 +12,9 @@
 
     <biggive-grid
       column-count="3">
-      <biggive-basic-card *ngIf="greenMatchApplicationsOpen"
+      <biggive-basic-card
         space-below='4'
+        heading-level = 2
```

Comment:
For consistency:
```suggestion
        heading-level=2
```

Reply:
@bdsl Also not sure but pls test locally that it works when passed as an int and not as a string.

Reply:
@alihejazi I have tested it locally, it works. The screenshot in the PR descriptions shows the issue goes away in Lighthouse.